test/util.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
+var sourceMapping = require('./resources/sourceMapping')
+var index = require('../src/index')
+var tape = require('tape')
+var util = require('../src/helpers/util')
+
+tape('Util', function (t) {
+  t.test('lowerbound', function (st) {
+    st.plan(6)
+    var array = [2, 5, 8, 9, 45, 56, 78]
```

Comment:
Please add a test case for an empty array and for an array that contains duplicates.